sdk/src/test/AllTests.ls

```diff
@@ -30,9 +30,11 @@ package {
      */
     public class AllTests {   
         public static function main() {
-            var result:TestResult = TestRunner.runAll((AllTests as Type).getAssembly());
-            
-            Process.exit(result.typeReport.successful ? 0 : 1);
+            TestRunner.onComplete += function(result:TestResult) {
+                
+                Process.exit(result.typeReport.successful ? 0 : 1);
+            };
+            TestRunner.runAll((AllTests as Type).getAssembly(), true);
```

Comment:
There are definitely ways of doing multiple asynchronous tests at the same time while keeping things static, but I am leery about doing that because it can make things very messy.

Anyway, doing the tests one at a time matches Mocha, which is the platform I am trying to emulate. This can be changed if need be, though